.github/workflows/gen_docs.yml

```diff
@@ -12,15 +12,24 @@ jobs:
       with:
         persist-credentials: false
         fetch-depth: 0
+
     - name: Gen Docs
       if: "!contains(github.event.head_commit.message, 'skip-ci')"
       run: make docker-compose
+
     - name: Commit files
       run: |
         git config --local user.email "syslbot@anz.com"
         git config --local user.name "SyslBot"
         git add .
         git commit -m "Regenerate docs [skip-ci]" && echo ::set-env name=can_push::1 || echo "No changes to docs"
+
+    - name: Auto approve
+      uses: hmarr/auto-approve-action@v2.0.0
+      if: env.can_push == 1
```

Comment:
Just curious if we need to restrict this to the sysl-bot user with something like `if: github.actor == 'dependabot[bot]' || github.actor == 'dependabot-preview[bot]'`